src/InfluxDB.Collector/InfluxDB.Collector.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>A minimal metrics collection API for InfluxDB</Description>
     <Authors>influxdb-csharp Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
I think a move to drop the old targets is probably worthwhile, given the additional maintenance/test burden of multi-targeting. We'd need to mark this as `breaking`, however, and rev the package to 2.0.